File: Jour_2/CallCenter.java
public class CallCenter {
    public void dispatchCall(Call call) {
        Respondent respondent = new Respondent();
        Manager manager = new Manager();
        Director director = new Director();


        if (respondent.isFree) {
           respondent.handleCall(call);
            System.out.println("Respondent handling call: " + respondent.name + respondent.employee);
        } else if (manager.isFree) {
             manager.handleCall(call);
            System.out.println("Manager handling call: "  + manager.name + manager.employee);
        } else {
          director.handleCall(call);
            System.out.println("Director handling call: "  + director.name + director.employee);
        }

    }



    public static void main(String[] args) {
        CallCenter callCenter = new CallCenter();
        Call incomingCall = new Call("John Doe");
        callCenter.dispatchCall(incomingCall);

    }




}

File: Jour_2/call_center.py
class Employees():
  #niveau hiérarchique des employé·e·s
  level = None
  #temps en seconde pour répondre avant que l’appel soit transmis
  handleTime = 60
  def __init__(self, employees, busy):
    self.employees = employees
    #mais comment passer de None à True ou False ?
    self.busy = False

  def answerCall(self):
  def escalateCall(self):


class Respondent(Employees):
  level = 0
  def __init__(self):
    super(Respondent, self).__init__(employees, busy)
    


class Manager(Employees):
  level = 1
  def __init__(self):
    super(Manager, self).__init__(employees, busy)

class Director(Employees):
  level = 2
  def __init__(self):
    super(Director, self).__init__(employees)

# method dispatchcall()
# create callCenter object ?
# if handleTime

class callCenter():
  
  def __init__(self):
    self.